pkg/duck/sinks.go

```diff
@@ -17,50 +17,77 @@ limitations under the License.
 package duck
 
 import (
-	"context"
 	"fmt"
 
 	corev1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/client-go/dynamic"
+	"k8s.io/apimachinery/pkg/api/meta"
 
-	duckapis "github.com/knative/pkg/apis"
-	"github.com/knative/pkg/apis/duck"
+	pkgapisduck "github.com/knative/pkg/apis/duck"
 	duckv1alpha1 "github.com/knative/pkg/apis/duck/v1alpha1"
+
+	"github.com/knative/eventing/pkg/reconciler"
+	"github.com/knative/pkg/tracker"
 )
 
-// GetSinkURI retrieves the sink URI from the object referenced by the given
-// ObjectReference.
-func GetSinkURI(ctx context.Context, dynamicClient dynamic.Interface, sink *corev1.ObjectReference, namespace string) (string, error) {
-	if sink == nil {
+// SinkReconciler is a helper for Sources. It triggers
+// reconciliation on creation of the source's sink.
```

Comment:
Is it only on creation, or any change of the source's sink?

Reply:
On all CRUD operations, as https://github.com/knative/eventing/blob/e647e96193e8af18008bb41d0a80b648fcad4721/pkg/reconciler/handler.go#L24 tells us.
Corrected the comment in 8b4eeaed99ef0cfc82ef8f66a1c1357c0a114ae4